src/components/pages/Offers/domain/isOfferFullyBooked.js

```diff
@@ -0,0 +1,9 @@
+export const isOfferFullyBooked = stocks => {
+  const hasUnlimitedStock = stocks.some(stock => stock.remainingQuantity === 'unlimited')
+
+  if (hasUnlimitedStock) {
+    return false
+  }
+
+  return stocks.every(stock => stock.remainingQuantity <= 0)
```

Comment:
```suggestion
  return !stocks.some(stock => stock.remainingQuantity > 0)
```
j'avais pas pensé qu'il fallait le ! au début, donc à voir si tu trouves ça pas clair

Reply:
J'avoue que je trouve le every plus clair 😄 c'est assez explicite, plutôt que d'avoir une négation de some qui va demander un peu + de boulot au cerveau (en tout cas au mien)

Reply:
:+1: